proto/viam/app/package/v1/package.proto

```diff
@@ -0,0 +1,103 @@
+syntax = "proto3";
+
+package viam.app.package.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "go.viam.com/api/app/package/v1";
+
+service PackageService {
+  // CreatePackage uploads a package's contents and information to the cloud
```

Comment:
I think this can just be `// CreatePackage uploads a package to the cloud`

Reply:
done!